### PR TITLE
Attempt to achieve #1

### LINKED ISSUE #1: Create Get: /manage/{id/url}
##### use already created functions:
-  store.GetAssetDetailsByUrl
-  store.GetAssetDetailsById

### PR DESCRIPTION
**Pull request by SnowballSH**
## Summary

1. Some small code cleaning because my IDE doesn't like the old code :( Feel free to tell me to undo it if you don't like the change.
2. Attempt to create endpoint /manage/url and /manage/id.

Example: `GET http://localhost:5000/manage/url/aa.png` => `{"id":6807691003781186055,"name":"bruh"}`

## Checklist

- [x] If endpoints were changed then they have been documented and tested.
    - [ ] I have updated the docmentation to reflect the changes.
    - [x] I have updated the tests to support the changes.
- [x] This PR fixes an issue.
- [x] This PR adds something new (e.g. new endpoint or parameter).
- [ ] This PR is a breaking change (e.g. endpoint or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by Shubhaankar-Sharma — APPROVED** [review]
Seems fine :), just edited the error message cause the m7md doesn't like caps and also go-lint doesn't like caps errors, waiting for GET: /manage/{id}

**Comment by Shubhaankar-Sharma** [comment]
Maybe we can try to improve coverage by checking errors in you handler?

**Comment by Shubhaankar-Sharma** [comment]
> Merging #5 (e957178) into main (fb8e815) will increase coverage by 1.23%.
 The diff coverage is 50.00%.


So basically, we covered only half of the lines you wrote and we need to achieve atleast 69 percent to get approved by codecov as 69% was the coverage before 👍

**Comment by SnowballSH** [comment]
Alright. Thanks sponge! I'll do that.

**Comment by SnowballSH** [comment]
> Maybe we can try to improve coverage by checking errors in you handler?

Ah right. I totally forgot to test the failure part 😂 Sorry about that and I'll do that soon

**Comment by SnowballSH** [comment]
I finished the two endpoints. The test coverage is too long that I have no idea if I did a good job covering. I added testing for not found and invalid id for my two endpoints.

**Comment by Shubhaankar-Sharma** [comment]
😂 i Get what you were saying in discord, I added creator id in the get url details query , when I saw that GetDetailsById was returning creator id...

File: api/handlers_test.go [hunk 1]
```diff
@@ -147,5 +147,56 @@ func TestCreateAsset(t *testing.T) {
...
+		checkResponseCode(t, http.StatusOK, fileRes.Code)
+		receivedFileData, err = ioutil.ReadAll(fileRes.Body)
+		require.NoError(t, err)
+
+		manageURLResponse := &db.GetAssetDetailsByUrlRow{}
+		err = json.Unmarshal(receivedFileData, manageURLResponse)
+		require.NoError(t, err)
+		require.Equal(t, assetId, int(manageURLResponse.ID))
+		require.Equal(t, assetName, manageURLResponse.Name)
+
```
Comment (Shubhaankar-Sharma) [inline api/handlers_test.go:166]: `require.Equal(t, AssetDetails.CreatorID, manageURLResponse.CreatorID)`

**Review by Shubhaankar-Sharma — CHANGES_REQUESTED** [review]
Yup just this needs to be add, I generated the sqlc query

**Comment by SnowballSH** [comment]
Alright, thanks! I'll do that soon.

**Comment by SnowballSH** [comment]
Should do now :)

**Review by Shubhaankar-Sharma — APPROVED** [review]

**Pull request merged**